### ISSUE TITLE
.eslintrc.js does not contain @typescript-eslint/parser

### ISSUE THREAD

**Issue by molvqingtai** [eslint/eslint#14680]
**Tell us about your environment**

Node version: v14.16.1
npm version: v6.14.12
Local ESLint version: v7.28.0 (Currently used)
Global ESLint version: Not found
Operating System: darwin 20.5.0

**What parser (default, `@babel/eslint-parser`, `@typescript-eslint/parser`, etc.) are you using?**

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```js
module.exports = {
  env: {
    browser: true,
    es2021: true
  },
  extends: [
    'standard'
  ],
  parserOptions: {
    ecmaVersion: 12,
    sourceType: 'module'
  },
  rules: {
  }
}

```

</details>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

When I run init and select typescript = Yes
```bash
# Does your project use TypeScript? Yes
npx eslint --init
```

**What did you expect to happen?**
`.eslintrc.js` should contain `@typescript-eslint/parser`

**What actually happened? Please copy-paste the actual, raw output from ESLint.**
`.eslintrc.js` does not contain `@typescript-eslint/parser`

**Steps to reproduce this issue:**

❯ npx eslint --init
✔ How would you like to use ESLint? · style
✔ What type of modules does your project use? · esm
✔ Which framework does your project use? · none
✔ Does your project use TypeScript? · No / Yes
✔ Where does your code run? · browser
✔ How would you like to define a style for your project? · guide
✔ Which style guide do you want to follow? · standard
✔ What format do you want your config file to be in? · JavaScript

**Are you willing to submit a pull request to fix this bug?**

**Comment by snitin315** [comment]
Hi @molvqingtai, I followed your steps and was unable to reproduce the issue.

The following configuration was generated in my case - 

<details>
<summary>Configuration</summary>


```js
module.exports = {
    "env": {
        "browser": true,
        "es2021": true
    },
    "extends": [
        "standard"
    ],
    "parser": "@typescript-eslint/parser",
    "parserOptions": {
        "ecmaVersion": 12,
        "sourceType": "module"
    },
    "plugins": [
        "@typescript-eslint"
    ],
    "rules": {
    }
};
```
</details>

As you can see above, it contains `@typescript-eslint/parser`.  Am I missing something here?

**Comment by molvqingtai** [comment]
I'm very sorry, but the contrast on my terminal is so low that I can't tell yes or no.
